### PR TITLE
[OSDOCS-12444]: Adding prereq and reference to disable HCP procedure

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.17+

Issue: https://issues.redhat.com/browse/OSDOCS-12444

Link to docs preview: https://84306--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-prepare/hcp-enable-disable.html#hcp-disable-feature_hcp-enable-disable

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by lahinson** [comment]
Hi @dtthuynh - When you can, PTAL at this small change to the hosted control planes docs. I moved the statement about uninstalling the HyperShift Operator to the prerequisites section to highlight the importance of that step. Hopefully, that should be clear enough. In addition, the procedure immediately preceding this one in the docs is "Uninstalling the HyperShift Operator", so customers should be able to find those steps easily.

**Review by dtthuynh — APPROVED** [review]
Awesome! Thank you @lahinson

File: modules/hcp-disable-feature.adoc [hunk 1]
```diff
@@ -5,7 +5,11 @@
 [id="hcp-disable-feature_{context}"]
 = Disabling the {hcp} feature
 
-Before you disable the {hcp} feature, you must first uninstall the HyperShift Operator. 
+To disable the {hcp} feature, complete the following steps.
```
Comment (dfitzmau) [inline modules/hcp-disable-feature.adoc:8]: If you need I would prefer to read something like "You can disable the {hcp} feature to improve the performance of your cluster."

**Review by dfitzmau — APPROVED** [review]

**Pull request merged**